test_data/text_test.segments

```diff
@@ -0,0 +1,5 @@
+file:C:\Users\Wafa\Documents\Downloader\test_data\text.txt-segment1
+**
+file:C:\Users\Wafa\Documents\Downloader\test_data\text.txt-segment2
+**
+file:C:\Users\Wafa\Documents\Downloader\fest_data\stext.txt-segment3
```

Comment:
I think the file path have typo - it should be test_data not fest_data, and text.txt not stext.txt